src/main/java/bio/terra/janitor/service/cleanup/flight/GoogleBucketCleanupStep.java

```diff
@@ -0,0 +1,36 @@
+package bio.terra.janitor.service.cleanup.flight;
+
+import bio.terra.cloudres.common.ClientConfig;
+import bio.terra.cloudres.google.storage.StorageCow;
+import bio.terra.janitor.db.TrackedResource;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import com.google.cloud.storage.StorageOptions;
+
+public class GoogleBucketCleanupStep implements Step {
+  private final StorageCow storageCow;
+
+  public GoogleBucketCleanupStep() {
+    // Janitor only uses CRL Cows to delete resources. Cleanup is not needed.
+    this.storageCow =
+        new StorageCow(
+            ClientConfig.Builder.newBuilder().setClient("terra-crl-janitor").build(),
+            StorageOptions.getDefaultInstance());
+  }
+
+  @Override
+  public StepResult doStep(FlightContext flightContext) {
+    TrackedResource trackedResource =
+        flightContext
+            .getInputParameters()
+            .get(FlightMapKeys.TRACKED_RESOURCE, TrackedResource.class);
+    storageCow.delete(trackedResource.cloudResourceUid().getGoogleBucketUid().getBucketName());
```

Comment:
To delete a bucket, we need to delete all the objects in the bucket. We can also leave a TODO for now. Let's start with a naive iteration over the blobs, deleting them individually with a `TODO: Set bucket lifetime to 0 and wait once Stairway supports waiting. This is more robust for buckets with many or very large objects.`

Reply:
Added a TODO